### PR TITLE
Correct IV order to Attack/Defense/Stamina

### PR DESCRIPTION
**Pull request by pki0**
## Description

As we talked today we would change IV order to Attack/Defense/Stamina to synch with all pages and apps.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Review by Obihoernchen — CHANGES_REQUESTED** [review]
Looks good, but gym page is missing.

**Review by Obihoernchen — APPROVED** [review]

**Comment by humer2000** [comment]
it would be great to have a working demo reflecting all the changes

**Comment by Obihoernchen** [comment]
@humer2000 here you go: https://vserver.obihoernchen.net/pokemon_dev/

File: core/js/trainer.content.js [hunk 1]
```diff
@@ -174,6 +164,16 @@ function printPokemon(pokemon,pokeimg_suffix){
...
+		$('<div>',
+			{
+			title: 'IV Stamina :'+pokemon.iv_stamina, 
+			class: 'progress-bar progress-bar-success' ,
+			role : 'progressbar',
+			'aria-valuenow' :pokemon.iv_stamina,
+			'aria-valuemin' : 0, 
+			'aria-valuemax' : 45
+		}).css('width',((100/45)*pokemon.iv_stamina ) + '%'))
+	progressBar.append(
```
Comment (Obihoernchen) [inline core/js/trainer.content.js]: This is a syntax error. `progressBar.append(` has to be above the `$('<div>',`

**Review by Obihoernchen — CHANGES_REQUESTED** [review]
Works expect a syntax error in trainer.content.js (see comment)

File: core/js/trainer.content.js (hunk 1 above)
Comment (pki0) [reply core/js/trainer.content.js]: Yes auf cause... should now be correct

**Review by Obihoernchen — APPROVED** [review]

**Pull request merged**